### DISCUSSION TITLE
Need help on reloading buefy/vue2 table

### DISCUSSION THREAD

**Discussion by fikoulli** [buefy/buefy#3763] [Q&A]
Hi there,

I need to simply reload a buefy table by code, after a row delete.

Plz help

"buefy": "^0.9.21", Vue2

**Comment by adamleemiller** [comment]
Just refresh the data from the database or use Javascript to remove the data from the data set. I assume your data is an array in which case just use Javascript.splice to remove the item. Vue is 100% reactive so the table will automatically update.
